### PR TITLE
Blank component template

### PR DESCRIPTION
**Pull request by Jimgskoop**
Created a blank component template JS file.

### CODE REVIEW

File: public/scripts/packages/ui/BlankTemplate.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+       
+     },
+     anotherMethod: function ( $super) {
+       // Your code here
+       // Call the parent method
+       $super();       
+     }
+   };
+
+})() ));
```
Comment (Iheartweb) [inline public/scripts/packages/ui/BlankTemplate.js:80]: This should be }() );

File: public/scripts/packages/ui/BlankTemplate.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+/**
+ * Representation of a BlankTemplate
+ * @class BlankTemplate
+ * @constructor
+ * @extends Abstract
+ * @param {HTMLElement} element The HTML element surrounded by the control
+ * @param {Object} settings Configuration properties for this instance
+ * @requires ptclass, Abstract
+ */ 
+BlankTemplate = Class.create( Abstract, ( (function(){
```
Comment (Iheartweb) [inline public/scripts/packages/ui/BlankTemplate.js:14]: this should be BlankTemplate = Class.create( Abstract, ( function() {

File: public/scripts/packages/ui/BlankTemplate.js [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+       // Call the parent's constructor
+       $super( $element, config );
+       
+       // Event bindings
+       $element.on( config.on, function ( event ) {
+        // Your code here
+       } );       
+       
+     },
+     anotherMethod: function ( $super) {
```
Comment (Iheartweb) [inline public/scripts/packages/ui/BlankTemplate.js:73]: Some spacing inconsistency: anotherMethod: function ( $super ) {

File: public/scripts/packages/ui/BlankTemplate.js [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+    * @type {Object}
+    */
+   config,
+   /**
+    * Instance of Button
+    * @property Button
+    * @type Object
+    * @private
+    */
+   self;
```
Comment (Iheartweb) [inline public/scripts/packages/ui/BlankTemplate.js:43]: Hmm, should self and config be here? I thought they were instance level.

Comment (Jimgskoop) [inline public/scripts/packages/ui/BlankTemplate.js:43]: Right, I'm moving them.

**Pull request merged**